pulseapi/entries/models.py

```diff
@@ -27,7 +46,18 @@ def public(self):
         """
         Return all entries to start
         """
-        return self.filter(is_approved=True)
+        try:
+            # This has to happen in a try/catch, so that migrations
```

Comment:
Does this mean that once the migration has been run on production, we can remove the try/catch aspect of this?